### PR TITLE
Update setup info

### PR DESCRIPTION
**Pull request by tomwijnroks**

### CODE REVIEW

File: DEV_SETUP.md [hunk 1]
```diff
@@ -19,6 +19,15 @@ The server at github pages that builds and hosts the techblog does not always ru
 ### Updating dependencies
 To keep your local dev environment in sync, a [github-pages ruby gem](https://rubygems.org/gems/github-pages) is used. It can be found in the `Gemfile`. This gem is updated when the dependency versions change. Run `bundle update` to check if a new version is available for this gem.
 
+### macOS Sierra
+Gem `nokogiri` fails to install on macOS Sierra. You can use [brew](http://brew.sh/) to build it with the non-system-libxml2:
+```
+brew install libxml2
+bundle config build.nokogiri --use-system-libraries --with-xml2-include=/usr/local/opt/libxml2/include/libxml2
+sudo gem install nokogiri -v '1.6.8.1'
```
Comment (styxit) [inline DEV_SETUP.md]: This will install a specific version, while the github pages gem manages the required versions.

Comment (tomwijnroks) [inline DEV_SETUP.md]: @styxit that is the version from the `github-pages` dependency versions page. Do you have a better suggestion?

Comment (styxit) [inline DEV_SETUP.md]: That is the current version yes, the reason this gem is used is so we do not have to maintain the versions ourself. I would prefer to manage this with bundler/gem.

Sorry, no time to figure this out now.

Comment (tomwijnroks) [inline DEV_SETUP.md]: Created a new pull request to explain that it is a temporary workaround.
https://github.com/exonet/exonet.github.io/pull/20

**Pull request merged**